deploy/olm-catalog/portworx/1.5.0/portworxoperator.v1.5.0.clusterserviceversion.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
     capabilities: Auto Pilot
     categories: "Storage"
     description: Cloud native storage solution for production workloads
-    containerImage: openstorage/openstorage-operator:master
+    containerImage: portworx/px-operator:1.5.0-dev
```

Comment:
Originally we used to push our dev images to `openstorage/openstorage-operator`. All our jenkins jobs use this image. Starting 1.5.0, I am changing it to use `portworx/px-operator` so we don't have to maintain the former at all.